WordPress/Classes/Utility/App Configuration/AppConfiguration.swift

```diff
@@ -1,12 +1,16 @@
 import Foundation
+import BuildSettingsKit
 
-/**
- * WordPress Configuration
- * - Warning:
- * This configuration class has a **Jetpack** counterpart in the Jetpack bundle.
- * Make sure to keep them in sync to avoid build errors when building the Jetpack target.
- */
+/// - warning: Soft-deprecated. Use `BuildSettings` directly.
 @objc class AppConfiguration: NSObject {
-    @objc static let isJetpack: Bool = false
-    @objc static let isWordPress: Bool = true
+    @objc static var isJetpack: Bool { app == .jetpack }
+    @objc static var isWordPress: Bool { app == .wordpress }
+
+    private static var app: AppBrand {
+        // TODO: remove when tests can be updated
+        if BuildSettingsEnvironment.current == .test {
```

Comment:
Not my proudest code, but there are too many unit that rely on `AppConfiguration` being available and being set to `.wordpress`.